assets/app/views/edit/build-config.html

```diff
@@ -0,0 +1,519 @@
+                              </a>
+                            </span>
+                          </label>
+                          <div>
+                            <input class="form-control"
+                              id="binaryAsBuild"
+                              name="binaryAsBuild"
+                              type="text"
+                              ng-model="options.binaryAsFile"
+                              autocorrect="off"
+                              autocapitalize="off"
+                              spellcheck="false">
+                          </div>
+                        </div>
+                      </div>
+
+                      <div class="form-groups" ng-show="sources.image">
+                        <div class="form-group">
+                          <label for="imageSourceFrom">Image Source From</label>
+                          <select 
+                            class="form-control simpleselector"
+                            ng-model="options.pickedImageSourceType"
+                            ng-options="type.id as type.title for type in imageSourceTypes"
+                            ng-change="assambleInputType('imageSource', options.pickedImageSourceType)">
+                          </select>
+                        </div>
+
+                        <div class="row form-group" ng-if="options.pickedImageSourceType=='ImageStreamTag'" >
+                          <div class="col-lg-4">
+                           <div class="label-with-selector">
+                              <label class="multiselector-label">Namespace</label>
+                              <select
+                                name="imageSourceNamespace"
+                                class="form-control multiselector select-imagesource-namespace"
+                                ng-model="options.pickedImageSourceNamespace"
+                                ng-options="name for name in imageSourceBuildFrom.projects track by name"
+                                ng-change="updateImageSourceImageStreams(options.pickedImageSourceNamespace, true)"
+                                required>
+                              </select>
+                              <div class="warnings">
+                                <span class="text-danger" ng-if="!availableProjects.contains(options.pickedImageSourceNamespace)">Selected Namespace is not available.</span>
```

Comment:
I'd probably just say "Could not load image streams from namespace." and not show the alert. It's odd to see the error twice. Perhaps add a tooltip with error details.